.github/workflows/pypi.yaml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
   push:
     branches:
-      - main
+      - master
```

Comment:
I recommend renaming the master branch to main though. But for now this will do.

Reply:
Agreed, let's do this for the next release.